packages/service-library/src/servicelib/fastapi/celery/app_server.py

```diff
@@ -18,7 +19,13 @@ def __init__(self, app: FastAPI):
         super().__init__(app)
         self._lifespan_manager: LifespanManager | None = None
 
-    async def lifespan(self, startup_completed_event: threading.Event) -> None:
+    @property
+    def task_manager(self) -> TaskManager:
+        assert self.app.state.task_manager, "Task manager is not initialized"  # nosec
+        task_manager: TaskManager = self.app.state.task_manager
+        return task_manager
+
+    async def start_and_hold(self, startup_completed_event: threading.Event) -> None:
```

Comment:
it is a lifespan and the one problem I see here is the returned type that is wrong. It should be `AsyncIterator[None]` which would remove the confusion I think